samples/subsys/mgmt/mcumgr/smp_svr/overlay-802154-subg.conf

```diff
@@ -0,0 +1,19 @@
+CONFIG_BT=n
+
```

Comment:
remove `overlay-` from the name

Reply:
I don't really mind, but why? This config has the same name in `echo_server`, `echo_client`, `coap_server`, `coap_client`, etc.